app/src/main/java/com/eveningoutpost/dexdrip/ui/MicroStatusImpl.java

```diff
@@ -28,12 +31,20 @@ public String gs(String id) {
     }
 
     @Override
-    public boolean bluetooth() {
-        return DexCollectionType.hasBluetooth();
+    public boolean bluetooth() { // Dexcom with Bluetooth except G7
+        return DexCollectionType.hasBluetooth() && !using_g7();
     }
 
     @Override
     public boolean xmitterBattery() {
         return DexCollectionType.usesClassicTransmitterBattery();
     }
+
+    @Override
+    public boolean using_g7() { // True if we are using G7
+        if (DexCollectionType.getDexCollectionType() == DexCollectionType.DexcomG5 && Pref.getBooleanDefaultFalse("using_g6") && shortTxId()) { // If using G7
+            return true;
+        }
+        return false;
```

Comment:
return DexCollectionType.getDexCollectionType() == DexCollectionType.DexcomG5 && Pref.getBooleanDefaultFalse("using_g6") && shortTxId()

Reply:
Thanks a lot.  Fixing it in a minute.